tests/receptor-tester.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
```

Comment:
I'm not a fan of the hand-rolled bash scripts. 

We should use a testing library, either Go or Python.

Reply:
But it's just a way to invoke commands. This is not `pytest` wrap or something like this, it's just a way to run the existent tests inside containers. Should I use `Python` CLI to invoke commands instead of `bash`, already designed for CLI invoking?

The `README.md` that I created maybe can give you a better idea why I created this script/tool.

The reason I created this is to avoid `Makefile`. `Makefile` are cool for a lot of stuff, but it lacks documentation. At least with this `bash` script I'm able to invoke regular commands like `Makefile` (which also uses `sh` or `bash`) with a documentation embedded.

Go applications implies a dedicated dir, build processes, and dependencies installed. Python also requires those stuff and even an "requirements.txt" file if I need to use some dependency to create a CLI.

I can recreate this using Go or Python, but the consequence will be a lot of complexity been introduced. I also can convert this to regular Makefile commands, but it lacks context and documentation. Should I do this anyway?